critiquebrainz/frontend/templates/entity_review.html

```diff
@@ -7,6 +7,8 @@
     {{ _('%(artist)s', artist = '<b>'|safe + entity.name | default(_('[Unknown artist]')) + '</b>'|safe) }}
   {% elif review.entity_type == 'label' %}
     {{ _('%(label)s', label = '<b>'|safe + entity.name | default(_('[Unknown label]')) + '</b>'|safe) }}
+  {% elif review.entity_type == 'recording' %}
+    {{ _('%(recording)s', recording = '<b>'|safe + entity.name | default(_('[Unknown recording]')) + '</b>'|safe) }}
```

Comment:
why is the bold tag inside the translated block? can't we just do 

    <b>{{entity.name}}</b> 

?
Additionally, I don't know why we're marking the entity name itself as translateable, that probably isn't required.
What is the "Unknown x" default for? How can you make a review for an entity that doesn't exist?

Reply:
Fixed. I have removed the Unknown defaults as well.